docs/business-finder-email-signup.md

```diff
@@ -0,0 +1,15 @@
+# EU Exit Readiness Business Finder Email signup
+
+Currently only the [EU Exit Business Readiness Finder](https://www.gov.uk/find-eu-exit-guidance-business) is making use of these features, but other finders could use these in the future.
+
+## Subscribing to multiple facet values
+
+Users can [pick and choose which `facet_values` ('links') to subscribe to](https://github.com/alphagov/finder-frontend/pull/1191), and will be emailed when content is tagged with _any of the links_ (provided the 'send email' checkbox is checked in content tagger). When already tagged content is updated, only major updates will trigger emails (as per normal email subscriptions).
```

Comment:
Can you link to the [code](https://github.com/alphagov/finder-frontend/blob/master/app/lib/email_alert_list_title_builder.rb), rather than the PR, which is long, and will get out of date.

This fragment: "provided the 'send email' checkbox is checked in content tagger" should be in content-tagger, if it isn't already.

I think this paragraph 

> and will be emailed when content is tagged with _any of the links_ (provided the 'send email' checkbox is checked in content tagger). When already tagged content is updated, only major updates will trigger emails (as per normal email subscriptions)

should be focus more on what's changed.

and will be emailed when content tagged to _any of the facets_  is published or has a major update.

Also links is an overloaded term and also applies to organisations, taxons etc. We're only talking about content tagged to links-based-facets, so I think facet is the most appropriate term to use here.